README.md

```diff
@@ -5,641 +5,291 @@
 
-// access aggregated information from total output, e.g.
-Integer tokens = totalOutput.getUsage().getCompletionTokens();
-System.out.println("Tokens: " + tokens);
+```java
+Destination destination = DestinationAccessor.getDestination("my-aicore");
+ApiClient client = Core.getClient(destination);
 ```
 
 </details>
 
-#### Spring Boot example
+---
 
-Please find [an example in our Spring Boot application](sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OpenAiController.java).
-It shows the usage of Spring Boot's `ResponseBodyEmitter` to stream the chat completion delta messages to the frontend in real-time.
+## Getting Started
 
-## Orchestration chat completion
+### What You'll Build
 
-### Prerequisites
+In this quickstart, you'll build a simple Java application that uses the OpenAI GPT-3.5 Turbo model for chat completion.
+The application will send a prompt to the AI model and display the generated response.
 
-- A deployed Orchestration service in AI Core.
-  - [Orchestration documentation](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration)
-  - <details>
-    <summary>An example orchestration deployment from the AI Core <code>/deployments</code> endpoint</summary>
-    <pre>
-    {
-      "id": "d123456abcdefg",
-      "deploymentUrl": "https://api.ai.intprod-eu12.eu-central-1.aws.ml.hana.ondemand.com/v2/inference/deployments/d123456abcdefg",
-      "configurationId": "12345-123-123-123-123456abcdefg",
-      "configurationName": "orchestration",
-      "scenarioId": "orchestration",
-      "status": "RUNNING",
-      "statusMessage": null,
-      "targetStatus": "RUNNING",
-      "lastOperation": "CREATE",
-      "latestRunningConfigurationId": "12345-123-123-123-123456abcdefg",
-      "ttl": null,
-      "createdAt": "2024-08-05T16:17:29Z",
-      "modifiedAt": "2024-08-06T06:32:50Z",
-      "submissionTime": "2024-08-05T16:17:40Z",
-      "startTime": "2024-08-05T16:18:41Z",
-      "completionTime": null
-    }
-    </pre>
-    </details>
-- A Java project with a Maven `pom.xml`
-  - Java 17 or higher
-  - Maven 3.9 or higher
-  - if Spring Boot is used, then minimum version 3
-- [Set the AI Core credentials as an environment variable for local testing](#set-credentials-as-dedicated-environment-variable)
-
-### Maven dependencies
-
-Add the following dependencies to your `pom.xml` file:
-
-```xml
-<dependencies>
-  <dependency>
-    <groupId>com.sap.ai.sdk</groupId>
-    <artifactId>orchestration</artifactId>
-    <version>${ai-sdk.version}</version>
-  </dependency>
-</dependencies>
-```
+### Prerequisites
 
-See [an example pom in our Spring Boot application](sample-code/spring-app/pom.xml)
+Before you begin, ensure you have:
 
-### Chat completion template
+- Met the [General Requirements](#general-requirements).
+- Set up the AI Core credentials
+  using [Option 1: Set Credentials as Environment Variable](#option-1-set-credentials-as-environment-variable).
```

Comment:
(Minor)

```suggestion
  using [(1) Environment Variable](#option-1-set-credentials-as-environment-variable) or [(2) Regular Service Binding](#option-1-set-credentials-as-environment-variable).
```

I would consider the second option also low effort and convenience enough to mention it here similarly to first option.